src/rpc/RPC.cpp

```diff
@@ -97,7 +97,8 @@ constexpr static WarningInfo warningInfos[]{
      "This is a clio server. clio only serves validated data. If you "
      "want to talk to rippled, include 'ledger_index':'current' in your "
      "request"},
-    {warnRPC_OUTDATED, "This server may be out of date"}};
+    {warnRPC_OUTDATED, "This server may be out of date"},
+    {warnRPC_RATE_LIMIT, "Your connection is approaching rate limit!"}};
```

Comment:
I vote for this message being "You are placing too much load on the server". @natenichols what do you think?

Reply:
I like something like
```
"Your connection is being rate limited"
```

Reply:
But I dont care about what we end up picking

Reply:
Keep in mind that this is a warning sent before the client is actually being rate limited

Reply:
How about then `"You are about to be rate limited"`

Reply:
Sounds good, I like it